tests/particle_interpolator_quadratic_least_squares_2d_solkz.prm

```diff
@@ -0,0 +1,125 @@
+# A test based on the SolKz benchmark that performs quadratic
+# least squares interpolation from particle properties to
+# compositional fields that are flagged as 'particle' advected fields.
+
```

Comment:
What is the difference between this test and the one you added above? The original difference for the linear least squares case was that one tests the addition of new particles, and the other the accurace of the interpolation. But for the quadratic least squares only the interpolation has changed, which would suggest we would only need one additional test. Or do you test something else with this test (or the other one)? If so, please add to the comment at the beginning of the files.

Reply:
Good catch! @sac-bsa  and I discussed this issue while he was creating the tests before making the PR.  He made the comment that the test without the addition of particles was unnecessary, since it was a duplication of the 'add particles', without testing the addition of particles. There may be something similar in the 3D bilinear interpolation tests.  We need to check. Mack mentioned that one of the current overarching goals is to make the tests finish sooner, or run faster.  Please accept my apologies for not removing the test that doesn't add particles, thereby making more work for you.